### ISSUE TITLE
Support Comfy custom MultiGPU Nodes

### ISSUE THREAD

**Issue by Blahblahdenver** [mcmonkeyprojects/SwarmUI#161]
### Your question

I am using SwarmUI with Comfy, I've downloaded the nodes from https://github.com/neuratech-ai/ComfyUI-MultiGPU.git
I have everything setup perfectly. on windows 10. rtx4090 and 2070 installed. all GPU are visible. Cuda 12.2. when I use the dropdown menu within the comfy workflow MultiGPU nodes only Cuda 0 shows up. not both Cuda 1 and 0. Someone suggested SwarmUI is preventing both from being found. i have read https://github.com/mcmonkeyprojects/SwarmUI/blob/master/docs/Using%20More%20GPUs.md and found the backend setting but it does not let you use all GPU or have an 'any' option. you can only specify GPU_ID 0 or 1. wondering if theres a way to bypass this and let Comfyui just use all?
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 536.25 Driver Version: 536.25 CUDA Version: 12.2 |
|-----------------------------------------+----------------------+----------------------+
| GPU Name TCC/WDDM | Bus-Id Disp.A | Volatile Uncorr. ECC |
| Fan Temp Perf Pwr:Usage/Cap | Memory-Usage | GPU-Util Compute M. |
| | | MIG M. |
|=========================================+======================+======================|
| 0 NVIDIA GeForce RTX 4090 WDDM | 00000000:0A:00.0 On | Off |
| 30% 31C P8 33W / 450W | 429MiB / 24564MiB | 1% Default |
| | | N/A |
+-----------------------------------------+----------------------+----------------------+
| 1 NVIDIA GeForce RTX 2070 WDDM | 00000000:0B:00.0 Off | N/A |
| 0% 42C P8 16W / 175W | 0MiB / 8192MiB | 0% Default |
| | | N/A |
+-----------------------------------------+----------------------+----------------------+
(base) PS C:\Windows\system32> python -c "import torch; [print(torch.cuda.get_device_name(i)) for i in range(torch.cuda.device_count())]"

NVIDIA GeForce RTX 4090
NVIDIA GeForce RTX 2070

**Comment by mcmonkey4eva** [comment]
I've updated the ComfyUI Self-Start Backend settings to treat the GPU_ID input as a string, and allow you to use cuda syntax, eg `0,1`, to provide the backend with multiple GPUs.

**Comment by Blahblahdenver** [comment]
> I've updated the ComfyUI Self-Start Backend settings to treat the GPU_ID input as a string, and allow you to use cuda syntax, eg `0,1`, to provide the backend with multiple GPUs.

You are the man! works flawlessly now. Thank you.

**Comment by FurkanGozukara** [comment]
@mcmonkey4eva does this mean now we dont need to add more and we can set all at once like 0,1,2,3,4 ?

**Comment by mcmonkey4eva** [comment]
no, this is for one comfy backend to recognize multiple GPUs (there's a few nodes that can take limited advantage of this), separately from the swarm feature to actually make full use of multiple gpus

**Comment by FurkanGozukara** [comment]
@mcmonkey4eva what i mean is it will use like adding each backend for each gpu? i mean it will make our job easy. i know it will still queue on each one

**Comment by mcmonkey4eva** [comment]
Nothing's changed about how to configure GPUs atm as long as you're not using those specific comfy node packs
